Classes/Plugin/Calendar.php

```diff
@@ -49,6 +49,43 @@ class Calendar extends \Kitodo\Dlf\Common\AbstractPlugin
      */
     public function main($content, $conf)
     {
+        $this->init($conf);
+
+        // Set initial document (anchor or year file) if configured.
+        if (empty($this->piVars['id']) && !empty($this->conf['initialDocument'])) {
+            $this->piVars['id'] = $this->conf['initialDocument'];
+        }
+
+        // Load current document.
+        $this->loadDocument();
+        if ($this->doc === null) {
+            // Quit without doing anything if required variables are not set.
+            return $content;
+        }
+
+        $metadata = $this->doc->getTitledata($this->doc->cPid);
+        if (!empty($metadata['type'][0])) {
+            // Calendar plugin does not support IIIF (yet). Abort for all newspaper related types.
+            if (
+                $doc instanceof IiifManifest
+                && array_search($metadata['type'][0], ['newspaper', 'ephemera', 'year', 'issue']) !== false
+            ) {
+                return $type;
```

Comment:
This doesn't make any sense. Why return anything when `$doc` is an `IiifManifest` which we don't support? And `$type` isn't defined anyways... Maybe a copy & paste error from `docTypeCheck`?

Reply:
This is just cut&copy code from `docTypeCheck`. Will send a new commit to fix this.